python/ray/data/_internal/execution/operators/base_physical_operator.py

```diff
@@ -115,7 +115,12 @@ def initialize_sub_progress_bars(self, position: int) -> int:
         if self._sub_progress_bar_names is not None:
             self._sub_progress_bar_dict = {}
             for name in self._sub_progress_bar_names:
-                bar = ProgressBar(name, self.num_outputs_total() or 1, position)
+                bar = ProgressBar(
+                    name,
+                    self.num_outputs_total() or 1,
+                    unit="block",
```

Comment:
sorry i missed this earlier. i think this is also `bundle` since `num_outputs_total()` returns bundles count
```suggestion
                    unit="bundle",
```

Reply:
You're right! Also missed it, thanks - just committed the changes 🙌